### PR TITLE
Add FAQ entry mentioning that Gruntwork Enterprise EKS is a subscription upgrade

### PR DESCRIPTION
**Pull request by yorinasub17**
Addresses https://gruntwork.atlassian.net/browse/WWW-128

### CODE REVIEW

File: pages/landing/enterprise-eks/pricing/index.html [hunk 1]
```diff
@@ -165,6 +165,12 @@
         Note that if you cancel in the first 30 days, you lose the license to the code in the Infrastructure as Code Library, so you must stop using it and delete any references to it; on the other hand, if you do stick with the annual subscription, and cancel after at least one year, you can keep using all the code you had been using up to that point.
       </p>
 
+  - question: What if I am already a Gruntwork customer?
+    answer: |
+      <p>
+        If you are already a subscriber to Gruntwork for AWS, you already have the same level of access as Gruntwork Standard. You can contact us using the form above to inquire about upgrading to Gruntwork Enterprise to get early access to Gruntwork Enterprise EKS features. Be sure to mention that you are a current subscriber on the contact form!
```
Comment (brikis98) [inline pages/landing/enterprise-eks/pricing/index.html:171]:
```suggestion
        If you are already a subscriber to Gruntwork for AWS, then you currently have access to "Gruntwork Standard" EKS features. The "Gruntwork Enterprise" EKS features are available as an upgrade / replacement for your current subscription. If you're interested in upgrading, please use the contact form to request early access, and be sure to mention that you are a current subscriber!
```

**Review by brikis98 — COMMENTED** [review]
Minor wording tweak and this is good to go.

**Comment by yorinasub17** [comment]
Suggestion applied! Can I get another review?

**Review by brikis98 — APPROVED** [review]
LGTM!

**Comment by yorinasub17** [comment]
Thanks for review! Merging now.

**Pull request merged**